package.json

```diff
@@ -52,6 +51,7 @@
     "@types/pretty-ms": "^4.0.0",
     "@types/sinon": "^7.0.12",
     "@types/tmp": "0.1.0",
+    "c8": "^7.0.0",
     "codecov": "^3.0.0",
```

Comment:
I assume codecov is no longer needed. Could this and codecov.yaml be removed?

Reply:
@nolanmar511 codecov is the bin used to upload coverage results to the upstream service we use, so this is still needed (nyc and c8 are what collect the coverage locally).

Reply:
Okay! Thanks for the clarification!